tensorflow/tools/ci_build/ci_test_on_install.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
Two more arguments for keeping my current approach: 
1) The command line "tensorflow/tools/ci_build/ci_test_on_install.sh GPU" is more succinct than "tensorflow/tools/ci_build/ci_build.sh GPU /tensorflow/tensorflow/tools/ci_build/builds/test_on_install.sh GPU" and avoids the redundancy in arguments. 
2) Putting install tests in a separate script buys us flexibility in case we need to change the way the build / install / tests process is done or adding additional step to the install test.